bugfix-components-button

dsamyliak opened the pull request:
Change color of box-shadow, now it`s working
Hover, focus change action to box-shadow: none

dsamyliak committed:
bugfix-components-button

Change color of box-shadow, now it`s working
Hover, focus change action to box-shadow: none

Sunwart commented on src/sass/components/_button.scss:
```diff
@@ -19,15 +19,19 @@
+    border 250ms cubic-bezier(0.4, 0, 0.2, 1), background-color 250ms cubic-bezier(0.4, 0, 0.2, 1),
+    box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1);
 }
 //   Tablet from 768px to 1439px
```
Дім, я тестувала - треба прибрати це медіа, бо через нього стає в 2 рядки. І зараз ширини, заданої на класі достатньо, щоб не розтягувалось на весь екран.

Sunwart reviewed (changes requested):
Прокоментувала

dsamyliak committed:
bugfix-components-button

Delete css:
@media screen and (min-width: 768px) {
  .btn__make-order {
    width: 277px;
  }

dsamyliak commented:
Галина, видалив медіа

Sunwart reviewed (approved):
+
